dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/common/UID.java

```diff
@@ -59,11 +60,22 @@ private UID( String value )
         this.value = value;
     }
 
+    @Override
+    public String toString()
+    {
+        return value;
```

Comment:
I guess you could also use lombok `@ToString` as well as `@EqualsAndHashCode`

Reply:
We are using `@EqualsAndHashCode` 👌🏻 

I tried to use @ToString but it will print `UID(value=JW6BrFd0HLu)` while I prefer `JW6BrFd0HLu`. So that we only insert the value in error messages instead of the value with its internal representation.